content/docs/testing/tips.md

```diff
@@ -81,3 +83,40 @@ spec:
 ```
 
 Note that CRDs created via the `crdDir` test suite configuration are available for use immediately and do not require an assert like this.
+
+# Helm testing
+
+You can test a Helm chart by installing it in either a test step or your test suite:
+
+```
+apiVersion: kudo.dev/v1alpha1
+kind: TestSuite
+commands:
+- command: kubectl create serviceaccount -n kube-system tiller
```

Comment:
Why not use the `kubectl` field for this?

Reply:
I used the imperative `kubectl create` commands for this example just because it keeps the example self-contained. I used `commands` here instead of `kubectl` since `kubectl create` will fail if the resources already exist and this allows me to set `ignoreFailure`. Mostly, `kubectl` is legacy since it doesn't support extra configuration.